integration-cli/docker_cli_events_test.go

```diff
@@ -628,7 +628,7 @@ func (s *DockerSuite) TestEventsCopy(c *check.C) {
 		c.Fatalf("Failed getting file:%q\n%q", out, err)
 	}
 
-	cmd = exec.Command(dockerBinary, "events", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
+	cmd = exec.Command(dockerBinary, "events", "--since=0", "-f", "container=cptest", "--until="+strconv.Itoa(int(since)))
```

Comment:
I think some clarity on what the default behavior is would be good. The docs (unless I missed it) don't seem to say what the default value for `--since` is. Is it "since time began" or is it "from now on" ?
Can we get a doc change to say which it is.

If its "since time began", since you added `--since=0` to all of theses, are we actually testing the default any more? The new test at the bottom would need to be modified to include some event prior to the `docker events` call.

If its "from now on" then that new test should be modified to include some docker commands that generates an event before the `docker events` call just to ensure that there's something in the event stream that will be skipped.